components/generator/iconpack/src/commonTest/kotlin/io/github/composegears/valkyrie/generator/iconpack/IconPackGeneratorTest.kt

```diff
@@ -22,29 +23,38 @@ class IconPackGeneratorTest {
 
     @Test
     fun `generate icon pack`() {
-        val result = IconPackGenerator.create(config = createConfig())
+        val result = createIconPackSpecOutput(config = createConfig())
 
-        val expected = getResourceText("iconpack/IconPack.kt")
+        val expected = """
```

Comment:
I will investigate if we can load resource in wasm. I tried https://github.com/goncalossilva/kotlinx-resources but it doesn't support necessary target.